index.js

```diff
@@ -449,6 +450,17 @@ module.exports = {
 						}
 						return ctx.succeed(result);
 					};
+
+					// adds values from withSessionAttributes to the session
+					if (currentItem.withSessionAttributes) {
+						var session = request.session.attributes;
+
+						for (var newAttribute in currentItem.withSessionAttributes) {
+							if (typeof session[`${newAttribute}`] === "undefined") {
+								session[`${newAttribute}`] = currentItem.withSessionAttributes[`${newAttribute}`];
```

Comment:
Why do you use template strings here?

Reply:
Template strings removed (it was an odd suggestion from a friend).